built_value/lib/built_value.dart

```diff
@@ -86,6 +86,11 @@ class BuiltValue {
 /// Fields marked with this annotation are allowed to be null.
 const String nullable = 'nullable';
 
+/// Memoized annotation for Built Value getters and methods.
+///
+/// Fields marked with this annotation are not compared in the == operator.
+const String ignored = 'ignored';
```

Comment:
I think the name should be more specific about what the field should be ignored for.
I mentioned ignore for serialization a while ago for example

Reply:
You're absolutely right! But, unfortunately I couldn't find any other word to describe the intent. Do you maybe have any suggestions?

Reply:
`ignoreInEquals`?

Reply:
Thanks for your suggestion @zoechi.
We went with `BuiltValueField(compare: false)` as suggested by @davidmorgan.